### ISSUE TITLE
Could you help me?

### ISSUE THREAD

**Issue by Mercury13** [mitradranirban/OpenNagMundari#2]
I’m the author of Unicodia, encyclopedia of Unicode characters. I’m drawing a simple but hand-drawn Nag.
Of course, I don’t know language, so if anyone knows, together we can make a working font, not just a sample for Unicodia.
![image](https://user-images.githubusercontent.com/10164882/194740077-91c931da-e3e2-44aa-93ff-bc97a4786965.png)

**Comment by mitradranirban** [comment]
I also do not know the language, but I know it is alphabetic, and like
Ol Chiki, and no complex font is required for rendering

**Comment by Mercury13** [comment]
I have already made characters up to E6.

**Comment by Mercury13** [comment]
Here’s my description of Nag — is it correct?

Non-written Austro-Asian (not Indo-European) language. According to 2011 census, 1.1 million people in India speak Mundari and 500 thousand more speak some Munda — it’s unknown whether it is the same language.

Writing systems are Odia, Devanagari, Bengali, Latin — but phonemes are exotic for Latin, and abugidas badly suit Austro-Asian languages. No wonder Rohidas Singh Nag invented a new alphabet in 1949, and it even took root. About 1982 he shortened the alphabet to 27 characters and showed to local authorities, and reformed in 2008.

Digits are own. Punctuation is European.

**Comment by mitradranirban** [comment]
I have created all the glyphs fot code points of NagMundari by scanning the unicode code table. You can check the glyph pictures in Glyphlist folder. My font contain all the code points of NagMundari, created by autotracing the glyph pictures.

**Comment by Mercury13** [comment]
I tried your font, that’s easier of course. But looks, excuse me, like hell, that’s why started to make my own.

**Comment by Mercury13** [comment]
![image](https://user-images.githubusercontent.com/10164882/194778028-7c45c300-5d61-4013-be9e-fbb0222a2f68.png)
Finally made, that’s enough for Unicodia. But maybe we’ll make a working font? — with correct tracking/kerning, laying marks and so on.

**Comment by Mercury13** [comment]
You don’t lay marks either, here’s your way of laying marks.
![image](https://user-images.githubusercontent.com/10164882/194778399-e2c99c7a-2a1d-4ad3-a04a-bbaad0655680.png)

**Comment by mitradranirban** [comment]
I also do not know the language, but I know it is alphabetic, and like
Ol Chiki, and no complex font is required for rendering

On 09/10/2022, Mercury13 ***@***.***> wrote:
> I’m the author of Unicodia, encyclopedia if Unicode characters. I’m drawing
> a simple but hand-drawn Nag.

> Of course, I don’t know language, so if anyone knows, together we can make a
> working font.

> ![image](https://user-images.githubusercontent.com/10164882/194740077-91c931da-e3e2-44aa-93ff-bc97a4786965.png)

>

>

> --

> Reply to this email directly or view it on GitHub:

> https://github.com/mitradranirban/OpenNagMundari/issues/2

> You are receiving this because you are subscribed to this thread.

>

> Message ID: ***@***.***>


-- 
Dr Anirban Mitra

**Comment by Mercury13** [comment]
![image](https://user-images.githubusercontent.com/10164882/195374784-dc148956-b3a0-46c9-9d57-96cf541f344c.png)
I changed NOTHING (got heavy flu, no desire to work), just added a small HTML test.
1. We’ve got a different approach to prime. I made it spacing, and yours is non-spacing. Which is correct?
2. I need to move umlauts further from line.
3. As umlauts are rare, my “default” placement of dots and tildes is rather acceptable.
4. Your line height and placement of umlauts is hell horrible.
5. My tracking initially was too tight, and afterwards I made it better.
6. If I find out how anchors work, I’ll put TR dot better.
7. Dislike curving of D5 (3 with tail), EA (2 with vertical stroke).
8. Replace tilde from Serif… at least with tilde from Sans!

**Comment by Mercury13** [comment]
9. What about your umlauts… upper dot and prime are too far, lower mid dot goes to the previous character, and tilde is OK. Font’s ascent is surely too low.

**Comment by Mercury13** [comment]
10. A special style of text was invented for homemade game typography engines that do not do tracking/kerning right (e.g. Supergiant Games’ “Bastion”): the letters are not aligned, and no matter how it tracks/kerns. This is your case.

**Comment by Mercury13** [comment]
Managed to track it somehow and hand-anchor lower umlaut:
![image](https://user-images.githubusercontent.com/10164882/195392405-b3423592-0736-45e7-932e-4e46644d61f5.png)

**Comment by Mercury13** [comment]
As I know now how to anchor marks, I hand-anchored top-right dot for a few characters:
![image](https://user-images.githubusercontent.com/10164882/195395402-9d5e1de9-93c6-4a38-87c9-b78118cff214.png)

**Comment by Mercury13** [comment]
No, prime is spacing, and it should probably look more like actual prime. So I’m right.
![image](https://user-images.githubusercontent.com/10164882/195399821-f0385361-1dae-47c0-9e55-6763d1ee2171.png)
![image](https://user-images.githubusercontent.com/10164882/195400044-77605355-a11d-4b80-a3a1-fe1a210f0101.png)

**Comment by Mercury13** [comment]
About prime: there are not so many post-reform primes, but it’s probably spacing. Pre-reform prime looked like actual mathematical prime.

**Comment by Mercury13** [comment]
What else? Moved prime upwards. Slight auto-kerning. RLIG E0+ED because of kerning glitches.

**Comment by Mercury13** [comment]
Go download what I’ve done, I have not finished yet!

**Comment by Mercury13** [comment]
So you see what makes a working font rather than collection of glyphs!

**Comment by Mercury13** [comment]
I stopped at six RLIGs right now, all relate to the case when the entire prime (more precisely, its advance width) is within main character’s advance width. It would be nice to turn a spacing prime to a mark, but IDK right now how to do.
Replaced and anchored tilde.

**Comment by Mercury13** [comment]
I won’t fix my Nag anymore, here’s what I’ve done.
![image](https://user-images.githubusercontent.com/10164882/195925777-cf01cb90-2bfe-4dfc-9c98-56c3670855a0.png)
